### PR TITLE
Extended readme for initial site setup

### PR DESCRIPTION
**Pull request by DuttonMa**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,20 +31,52 @@ Or, if available, you may also fetch the database and import this:
...
+which may be done by exporting config (land drush config-export). You will now see lots of migration Yaml files (starting
+   with 'migrate_plus...') in your config/<sitename> directory.
+
+5. Create a new custom module for your site. An example of this is web/sites/uregni/modules/custom/uregni_migrations. Your new module
+should have a config/install directory.
+
+6. Move the migrations that you exported from the config/<sitenam> directory into the config/install directory of your new module using
+commands like this:
+   mv migrate_plus.migration.* ../../../web/sites/odscni/modules/custom/odscni_migrations/config/install
+   mv migrate_plus.migration_group.migrate_*.yml ../../../web/sites/odscni/modules/custom/odscni_migrations/config/install
```
Comment (omahm) [inline README.md:71]: Should be able to move those with one command 'mv migrate_plus.migration*'

**Review by omahm — COMMENTED** [review]
Looks ok, just 1 comment

File: README.md (hunk 1 above)
Comment (DuttonMa) [reply README.md]: Yes, good spot, changing that now

**Review by omahm — APPROVED** [review]
👍

**Pull request merged**